source/manual/redirect-routes.html.md.erb

```diff
@@ -7,17 +7,131 @@ parent: "/manual.html"
 related_applications: [short-url-manager]
 ---
 
-Sometimes, there is a need to manually redirect existing URLs to another
-internal location, usually due to content being archived or because the
-slug has been changed.
+There is a regular requirement for creating short URLs on behalf of
+departments, usually for use with for campaign materials. Content
+designers do this using the [Short URL Manager][short-url-manager] app.
+Read [more about Short URL Manager][short-url-manager-section] below,
+including how to remove routes originating from it.
 
-There is also a regular requirement for creating short URLs on behalf of
-departments, usually for use with for campaign materials.
+There are also processes for [redirecting HRMC manuals][hmrc-manuals-section]
+and [redirecting from campaign sites][campaign-sites-section].
 
-## Getting information on current redirects
+Finally, sometimes there is a need to manually redirect existing URLs to
+another internal location, either due to an incident, content being
+archived or because the slug has been changed.
 
-To find if there is already a redirect for a particular path, open a router-api
-console:
+## Choosing a redirect strategy
+
+Does something exist at the incoming URL already? There's a
+[simple way of checking][check-route-exists-section].
+
+If no route exists, and if you have the time to wait for a content
+designer to review the proposal, you should just use Short URL Manager.
```

Comment:
I think we should discourage this if the route isn't what we could consider to be a "short URL", for example `gov.uk/dfe`.